providers/fipsinstall.cnf

```diff
@@ -0,0 +1,6 @@
+[fips_sect]
```

Comment:
This seems to be here in error?

Reply:
Yes, the file was renamed in #11177 which happened in April.  Lucky that this was the only rebase issue.